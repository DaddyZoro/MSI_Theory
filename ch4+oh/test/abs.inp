!***************************************************
!               GLOBAL SECTION
!***************************************************
TemperatureList[K]                      300 400 500 600 700 800 900 1000 1100 1200 1300 1400 1500 1600 1700 1800 1900 2000 2100 2200 2300 2400 2500  
ModelEnergyLimit[kcal/mol]              400        ! Highest reference energy used in the calculation ( or ReferenceEnergy[kcal/mol])
RateOutput                              abs.out                        ! output file name for rate coefficients                         
!***************************************************
!***************************************************
!  REACTANTS					
!***************************************************
 Bimolecular REACS
 Fragment REACT1
 RRHO     
 Geometry[angstrom]               5
  C         0.000000000000      0.000000000000      0.000000000000          
  H         0.000000000000      0.000000000000      1.086867923611          
  H         1.024668348555      0.000000000000     -0.362560113663          
  H        -0.512368490410     -0.887418550959     -0.362442424276          
  H        -0.512368490412      0.887418550957     -0.362442424277          
 	Core 	RigidRotor
 		SymmetryFactor    12.0000000000000     
 	End
     Frequencies[1/cm]            9
   1339.3      1340.5      1340.5      1564.9      1564.9      3065.9      3179.1      3179.5      3180.3    
  ZeroEnergy[kcal/mol]             0.
  ElectronicLevels[1/cm]                      1
  0.000000000000000E+000   1.00000000000000     
 End 
 !************************************
 Fragment REACT2
 RRHO     
 Geometry[angstrom]               2
  O         0.000000000000      0.000000000000      0.000000000000          
  H         0.000000000000      0.000000000000      0.971429073414          
 	Core 	RigidRotor
 		SymmetryFactor    1.00000000000000     
 	End
     Frequencies[1/cm]            1
   3773.5    
  ZeroEnergy[kcal/mol]             0.
  ElectronicLevels[1/cm]                      2
  0.000000000000000E+000   2.00000000000000     
   140.000000000000        2.00000000000000     
 End 
 GroundEnergy[kcal/mol] 0.0
 End
 !************************************
 Bimolecular PRODS
 Fragment PROD1
 RRHO     
 Geometry[angstrom]               4
  C         0.000000000000      0.000000000000      0.000000000000          
  H         0.000000000000      0.000000000000      1.076304113129          
  H         0.932099834210      0.000000000000     -0.538146331156          
  H        -0.932099834210      0.000000000000     -0.538146331155          
 	Core 	RigidRotor
 		SymmetryFactor    6.00000000000000     
 	End
     Frequencies[1/cm]            6
   417.62      1412.1      1413.1      3144.9      3323.3      3324.5    
  ZeroEnergy[kcal/mol]             0.
  ElectronicLevels[1/cm]                      1
  0.000000000000000E+000   2.00000000000000     
 End 
 !************************************
 Fragment PROD2
 RRHO     
 Geometry[angstrom]               3
  O         0.000000000000      0.000000000000      0.000000000000          
  H         0.000000000000      0.000000000000      0.958978247668          
  H         0.926825510355      0.000000000000     -0.246242014010          
 	Core 	RigidRotor
 		SymmetryFactor    2.00000000000000     
 	End
     Frequencies[1/cm]            3
   1626.3      3872.9      3976.0    
  ZeroEnergy[kcal/mol]             0.
  ElectronicLevels[1/cm]                      1
  0.000000000000000E+000   1.00000000000000     
 End 
 GroundEnergy[kcal/mol]   -14.79 
 End
 !************************************
 Barrier B2 Reacs Prods
 Variational
 RRHO            !            11
 Geometry[angstrom]            7
  C   -1.23226   0.12323   0.43196                           
  H   -1.25258   0.12159   1.51757                           
  H   -0.12722   0.13903   0.14168                           
  H   -1.69612  -0.76923   0.02336                           
  H   -1.66895   1.02702   0.01843                           
  O    1.16342  -0.07237  -0.37257                           
  H    0.95288  -0.83703  -0.93144                           
 	Core 	RigidRotor
           SymmetryFactor    1.00000000000000     
 End
  Rotor                             Hindered                 
  Group    7                                                 
  Axis             6           3                             
  Symmetry            3                                      
  Potential[kcal/mol]   4
    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00
  End                                                        
     Frequencies[1/cm]           13
   3791.8      3205.9      3202.2      3092.8      1720.8      1487.8      1479.6      1343.4      1223.4      1206.4    
   801.52      212.74      26.120    
 ZeroEnergy[kcal/mol]    5.41216340511141     
  ElectronicLevels[1/cm]                      1
  0.000000000000000E+000   2.00000000000000     
 End
 !*********************************************
 RRHO            !            12
 Geometry[angstrom]            7
  C   -1.23078   0.12295   0.43135                           
  H   -1.24698   0.12179   1.51694                           
  H   -0.11925   0.13535   0.13673                           
  H   -1.69469  -0.76944   0.02289                           
  H   -1.66411   1.02782   0.01686                           
  O    1.16103  -0.07201  -0.37164                           
  H    0.95328  -0.83669  -0.93131                           
 	Core 	RigidRotor
           SymmetryFactor    1.00000000000000     
 End
  Rotor                             Hindered                 
  Group    7                                                 
  Axis             6           3                             
  Symmetry            3                                      
  Potential[kcal/mol]   4
    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00
  End                                                        
     Frequencies[1/cm]           13
   3793.2      3207.7      3203.8      3092.9      1618.2      1482.0      1476.4      1343.8      1219.4      1179.6    
   814.38      236.16      121.89    
 ZeroEnergy[kcal/mol]    5.30176389691867     
  ElectronicLevels[1/cm]                      1
  0.000000000000000E+000   2.00000000000000     
 End
 !*********************************************
 RRHO            !            13
 Geometry[angstrom]            7
  C   -1.22951   0.12272   0.43083                           
  H   -1.24155   0.12195   1.51637                           
  H   -0.11015   0.13176   0.13155                           
  H   -1.69306  -0.76974   0.02231                           
  H   -1.65942   1.02862   0.01528                           
  O    1.15874  -0.07167  -0.37075                           
  H    0.95374  -0.83634  -0.93117                           
 	Core 	RigidRotor
           SymmetryFactor    1.00000000000000     
 End
  Rotor                             Hindered                 
  Group    7                                                 
  Axis             6           3                             
  Symmetry            3                                      
  Potential[kcal/mol]   4
    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00
  End                                                        
     Frequencies[1/cm]           13
   3794.9      3209.7      3205.6      3093.1      1527.9      1473.4      1466.8      1343.8      1214.7      1136.0    
   827.00      260.46      176.39    
 ZeroEnergy[kcal/mol]    5.25673942963232     
  ElectronicLevels[1/cm]                      1
  0.000000000000000E+000   2.00000000000000     
 End
 !*********************************************
 RRHO            !            14
 Geometry[angstrom]            7
  C   -1.22853   0.12252   0.43040                           
  H   -1.23640   0.12207   1.51587                           
  H   -0.09981   0.12832   0.12613                           
  H   -1.69130  -0.77012   0.02165                           
  H   -1.65497   1.02940   0.01371                           
  O    1.15660  -0.07136  -0.36993                           
  H    0.95426  -0.83599  -0.93105                           
 	Core 	RigidRotor
           SymmetryFactor    1.00000000000000     
 End
  Rotor                             Hindered                 
  Group    7                                                 
  Axis             6           3                             
  Symmetry            3                                      
  Potential[kcal/mol]   4
    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00
  End                                                        
     Frequencies[1/cm]           13
   3796.9      3211.8      3207.5      3093.3      1491.6      1469.6      1418.8      1340.8      1209.0      1066.6    
   838.02      285.20      222.21    
 ZeroEnergy[kcal/mol]    5.20920036673805     
  ElectronicLevels[1/cm]                      1
  0.000000000000000E+000   2.00000000000000     
 End
 !*********************************************
 RRHO            !            15
 Geometry[angstrom]            7
  C   -1.22784   0.12234   0.43009                           
  H   -1.23159   0.12221   1.51542                           
  H   -0.08822   0.12504   0.12052                           
  H   -1.68946  -0.77053   0.02096                           
  H   -1.65087   1.03017   0.01220                           
  O    1.15464  -0.07107  -0.36918                           
  H    0.95480  -0.83563  -0.93092                           
 	Core 	RigidRotor
           SymmetryFactor    1.00000000000000     
 End
  Rotor                             Hindered                 
  Group    7                                                 
  Axis             6           3                             
  Symmetry            3                                      
  Potential[kcal/mol]   4
    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00
  End                                                        
     Frequencies[1/cm]           13
   3799.4      3214.1      3210.0      3093.8      1482.4      1465.7      1381.7      1324.4      1201.9      974.78    
   842.35      309.15      263.02    
 ZeroEnergy[kcal/mol]    5.12352178052377     
  ElectronicLevels[1/cm]                      1
  0.000000000000000E+000   2.00000000000000     
 End
 !*********************************************
 RRHO            !            16
 Geometry[angstrom]            7
   C         0.000000000000      0.000000000000      0.000000
   H         0.000000000000      0.000000000000      1.085235
   H         1.152507724162      0.000000000000     -0.315114
   H        -0.460123588112     -0.893271832486     -0.409743
   H        -0.419813440672      0.908589343733     -0.419043
   O         2.380599002360     -0.193085212639     -0.798496
   H         2.183039149294     -0.957533106268     -1.360773
 	Core 	RigidRotor
           SymmetryFactor    1.00000000000000     
 End
  Rotor                             Hindered                 
  Group    7                                                 
  Axis             6           3                             
  Symmetry            3                                      
  Potential[kcal/mol]   4
    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00
  End                                                        
     Frequencies[1/cm]           13
   3802.3      3216.6      3212.3      3094.3      1476.7      1461.8      1374.8      1293.0      1192.9      893.64    
   771.99      332.52      300.79    
 ZeroEnergy[kcal/mol]    4.93976317866429     
  ElectronicLevels[1/cm]                      1
  0.000000000000000E+000   2.00000000000000     
 End
 !*********************************************
 RRHO            !            17
 Geometry[angstrom]            7
  C   -1.22729   0.12212   0.42976                           
  H   -1.22354   0.12226   1.51483                           
  H   -0.06191   0.11942   0.10910                           
  H   -1.68587  -0.77148   0.01943                           
  H   -1.64391   1.03152   0.00954                           
  O    1.15132  -0.07062  -0.36793                           
  H    0.95604  -0.83494  -0.93073                           
 	Core 	RigidRotor
           SymmetryFactor    1.00000000000000     
 End
  Rotor                             Hindered                 
  Group    7                                                 
  Axis             6           3                             
  Symmetry            3                                      
  Potential[kcal/mol]   4
    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00
  End                                                        
     Frequencies[1/cm]           13
   3805.0      3219.0      3214.6      3094.9      1473.1      1458.1      1380.6      1278.6      1182.0      897.24    
   748.38      351.62      330.64    
 ZeroEnergy[kcal/mol]    4.93926500453256     
  ElectronicLevels[1/cm]                      1
  0.000000000000000E+000   2.00000000000000     
 End
 !*********************************************
 RRHO            !            18
 Geometry[angstrom]            7
  C   -1.22733   0.12204   0.42971                           
  H   -1.22015   0.12219   1.51466                           
  H   -0.04777   0.11699   0.10340                           
  H   -1.68421  -0.77199   0.01864                           
  H   -1.64093   1.03211   0.00839                           
  O    1.14991  -0.07043  -0.36740                           
  H    0.95676  -0.83461  -0.93068                           
 	Core 	RigidRotor
           SymmetryFactor    1.00000000000000     
 End
  Rotor                             Hindered                 
  Group    7                                                 
  Axis             6           3                             
  Symmetry            3                                      
  Potential[kcal/mol]   4
    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00
  End                                                        
     Frequencies[1/cm]           13
   3807.9      3221.4      3216.5      3095.3      1469.8      1454.5      1387.8      1261.9      1169.1      898.65    
   686.09      369.33      357.24    
 ZeroEnergy[kcal/mol]    4.83997275421213     
  ElectronicLevels[1/cm]                      1
  0.000000000000000E+000   2.00000000000000     
 End
 !*********************************************
 RRHO            !            19
 Geometry[angstrom]            7
  C   -1.22751   0.12198   0.42971                           
  H   -1.21704   0.12218   1.51450                           
  H   -0.03327   0.11470   0.09771                           
  H   -1.68262  -0.77246   0.01791                           
  H   -1.63824   1.03269   0.00731                           
  O    1.14862  -0.07027  -0.36692                           
  H    0.95746  -0.83428  -0.93061                           
 	Core 	RigidRotor
           SymmetryFactor    1.00000000000000     
 End
  Rotor                             Hindered                 
  Group    7                                                 
  Axis             6           3                             
  Symmetry            3                                      
  Potential[kcal/mol]   4
    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00
  End                                                        
     Frequencies[1/cm]           13
   3811.1      3223.7      3218.8      3095.9      1467.5      1451.1      1396.8      1247.5      1154.3      900.00    
   640.17      384.52      379.71    
 ZeroEnergy[kcal/mol]    4.70193134200302     
  ElectronicLevels[1/cm]                      1
  0.000000000000000E+000   2.00000000000000     
 End
 !*********************************************
 RRHO            !            20
 Geometry[angstrom]            7
  C   -1.22781   0.12193   0.42976                           
  H   -1.21420   0.12216   1.51437                           
  H   -0.01849   0.11256   0.09206                           
  H   -1.68111  -0.77293   0.01720                           
  H   -1.63578   1.03323   0.00630                           
  O    1.14744  -0.07012  -0.36649                           
  H    0.95817  -0.83395  -0.93055                           
 	Core 	RigidRotor
           SymmetryFactor    1.00000000000000     
 End
  Rotor                             Hindered                 
  Group    7                                                 
  Axis             6           3                             
  Symmetry            3                                      
  Potential[kcal/mol]   4
    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00
  End                                                        
     Frequencies[1/cm]           13
   3814.3      3226.0      3220.9      3096.5      1466.6      1447.9      1406.6      1234.7      1138.0      898.97    
   605.52      398.81      397.01    
 ZeroEnergy[kcal/mol]    4.51583425589060     
  ElectronicLevels[1/cm]                      1
  0.000000000000000E+000   2.00000000000000     
 End
 !*********************************************
 RRHO            !            21
 Geometry[angstrom]            7
  C   -1.22820   0.12189   0.42984                           
  H   -1.21156   0.12214   1.51427                           
  H   -0.00350   0.11056   0.08647                           
  H   -1.67966  -0.77339   0.01652                           
  H   -1.63351   1.03375   0.00535                           
  O    1.14634  -0.06999  -0.36609                           
  H    0.95891  -0.83363  -0.93049                           
 	Core 	RigidRotor
           SymmetryFactor    1.00000000000000     
 End
  Rotor                             Hindered                 
  Group    7                                                 
  Axis             6           3                             
  Symmetry            3                                      
  Potential[kcal/mol]   4
    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00
  End                                                        
     Frequencies[1/cm]           13
   3817.6      3228.2      3223.0      3097.1      1467.8      1444.9      1416.0      1223.6      1120.1      895.21    
   581.11      414.49      406.88    
 ZeroEnergy[kcal/mol]    4.26602913599427     
  ElectronicLevels[1/cm]                      1
  0.000000000000000E+000   2.00000000000000     
 End
 !*********************************************
 RRHO            !            22
 Geometry[angstrom]            7
  C   -1.22867   0.12186   0.42994                           
  H   -1.20909   0.12212   1.51419                           
  H    0.01165   0.10867   0.08091                           
  H   -1.67826  -0.77385   0.01586                           
  H   -1.63138   1.03425   0.00445                           
  O    1.14532  -0.06987  -0.36572                           
  H    0.95966  -0.83331  -0.93045                           
 	Core 	RigidRotor
           SymmetryFactor    1.00000000000000     
 End
  Rotor                             Hindered                 
  Group    7                                                 
  Axis             6           3                             
  Symmetry            3                                      
  Potential[kcal/mol]   4
    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00
  End                                                        
     Frequencies[1/cm]           13
   3821.0      3230.2      3225.0      3097.6      1472.5      1442.2      1423.7      1213.9      1100.7      888.70    
   564.86      426.42      414.66    
 ZeroEnergy[kcal/mol]    3.94934233305561     
  ElectronicLevels[1/cm]                      1
  0.000000000000000E+000   2.00000000000000     
 End
 !*********************************************
 RRHO            !            23
 Geometry[angstrom]            7
  C   -1.22921   0.12184   0.43007                           
  H   -1.20674   0.12209   1.51414                           
  H    0.02690   0.10688   0.07540                           
  H   -1.67690  -0.77431   0.01520                           
  H   -1.62936   1.03473   0.00359                           
  O    1.14435  -0.06976  -0.36537                           
  H    0.96045  -0.83299  -0.93041                           
 	Core 	RigidRotor
           SymmetryFactor    1.00000000000000     
 End
  Rotor                             Hindered                 
  Group    7                                                 
  Axis             6           3                             
  Symmetry            3                                      
  Potential[kcal/mol]   4
    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00
  End                                                        
     Frequencies[1/cm]           13
   3824.4      3232.2      3226.9      3098.0      1482.2      1439.7      1428.5      1205.2      1079.5      879.39    
   552.54      434.66      420.82    
 ZeroEnergy[kcal/mol]    3.55982682864433     
  ElectronicLevels[1/cm]                      1
  0.000000000000000E+000   2.00000000000000     
 End
 !*********************************************
 RRHO            !            24
 Geometry[angstrom]            7
  C   -1.22982   0.12182   0.43023                           
  H   -1.20449   0.12207   1.51409                           
  H    0.04223   0.10517   0.06993                           
  H   -1.67555  -0.77477   0.01455                           
  H   -1.62743   1.03520   0.00275                           
  O    1.14344  -0.06966  -0.36505                           
  H    0.96127  -0.83268  -0.93038                           
 	Core 	RigidRotor
           SymmetryFactor    1.00000000000000     
 End
  Rotor                             Hindered                 
  Group    7                                                 
  Axis             6           3                             
  Symmetry            3                                      
  Potential[kcal/mol]   4
    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00
  End                                                        
     Frequencies[1/cm]           13
   3827.9      3234.2      3228.8      3098.4      1496.3      1437.4      1430.5      1197.2      1057.0      867.31    
   541.72      439.98      425.37    
 ZeroEnergy[kcal/mol]    3.08840556754937     
  ElectronicLevels[1/cm]                      1
  0.000000000000000E+000   2.00000000000000     
 End
 !*********************************************
 RRHO            !            25
 Geometry[angstrom]            7
  C   -1.23047   0.12181   0.43040                           
  H   -1.20230   0.12205   1.51405                           
  H    0.05763   0.10354   0.06451                           
  H   -1.67422  -0.77523   0.01390                           
  H   -1.62555   1.03566   0.00193                           
  O    1.14257  -0.06957  -0.36474                           
  H    0.96214  -0.83236  -0.93036                           
 	Core 	RigidRotor
           SymmetryFactor    1.00000000000000     
 End
  Rotor                             Hindered                 
  Group    7                                                 
  Axis             6           3                             
  Symmetry            3                                      
  Potential[kcal/mol]   4
    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00
  End                                                        
     Frequencies[1/cm]           13
   3831.5      3236.0      3230.6      3098.8      1513.0      1435.3      1430.9      1190.5      1033.4      852.59    
   532.76      442.74      428.53    
 ZeroEnergy[kcal/mol]    2.53650812150877     
  ElectronicLevels[1/cm]                      1
  0.000000000000000E+000   2.00000000000000     
 End
 !*********************************************
 RRHO            !            26
 Geometry[angstrom]            7
  C   -1.23118   0.12181   0.43059                           
  H   -1.20017   0.12203   1.51402                           
  H    0.07307   0.10200   0.05913                           
  H   -1.67289  -0.77569   0.01326                           
  H   -1.62372   1.03612   0.00113                           
  O    1.14174  -0.06948  -0.36445                           
  H    0.96304  -0.83204  -0.93035                           
 	Core 	RigidRotor
           SymmetryFactor    1.00000000000000     
 End
  Rotor                             Hindered                 
  Group    7                                                 
  Axis             6           3                             
  Symmetry            3                                      
  Potential[kcal/mol]   4
    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00
  End                                                        
     Frequencies[1/cm]           13
   3835.1      3237.9      3232.4      3099.2      1531.0      1433.4      1430.3      1185.6      1007.7      835.34    
   526.33      442.18      430.37    
 ZeroEnergy[kcal/mol]    1.91519792512397     
  ElectronicLevels[1/cm]                      1
  0.000000000000000E+000   2.00000000000000     
 End
 !*********************************************
 RRHO            !            27
 Geometry[angstrom]            7
  C   -1.23195   0.12181   0.43080                           
  H   -1.19808   0.12200   1.51399                           
  H    0.08855   0.10053   0.05381                           
  H   -1.67156  -0.77615   0.01261                           
  H   -1.62191   1.03657   0.00034                           
  O    1.14095  -0.06941  -0.36418                           
  H    0.96400  -0.83173  -0.93036                           
 	Core 	RigidRotor
           SymmetryFactor    1.00000000000000     
 End
  Rotor                             Hindered                 
  Group    7                                                 
  Axis             6           3                             
  Symmetry            3                                      
  Potential[kcal/mol]   4
    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00
  End                                                        
     Frequencies[1/cm]           13
   3838.9      3239.7      3234.1      3099.5      1549.9      1431.6      1429.4      1182.9      978.70      815.63    
   522.21      438.00      430.12    
 ZeroEnergy[kcal/mol]    1.22171590494052     
  ElectronicLevels[1/cm]                      1
  0.000000000000000E+000   2.00000000000000     
 End
 !*********************************************
 RRHO            !            28
 Geometry[angstrom]            7
  C   -1.23279   0.12181   0.43103                           
  H   -1.19600   0.12198   1.51396                           
  H    0.10405   0.09915   0.04855                           
  H   -1.67020  -0.77662   0.01195                           
  H   -1.62012   1.03702  -0.00045                           
  O    1.14020  -0.06934  -0.36393                           
  H    0.96501  -0.83142  -0.93038                           
 	Core 	RigidRotor
           SymmetryFactor    1.00000000000000     
 End
  Rotor                             Hindered                 
  Group    7                                                 
  Axis             6           3                             
  Symmetry            3                                      
  Potential[kcal/mol]   4
    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00
  End                                                        
     Frequencies[1/cm]           13
   3842.8      3241.5      3235.9      3099.9      1569.7      1430.1      1428.3      1183.0      946.62      793.57    
   520.21      431.67      426.15    
 ZeroEnergy[kcal/mol]   0.469112600275722     
  ElectronicLevels[1/cm]                      1
  0.000000000000000E+000   2.00000000000000     
 End
 !*********************************************
 RRHO            !            29
 Geometry[angstrom]            7
  C   -1.23368   0.12182   0.43128                           
  H   -1.19391   0.12196   1.51393                           
  H    0.11956   0.09786   0.04336                           
  H   -1.66881  -0.77709   0.01128                           
  H   -1.61832   1.03747  -0.00124                           
  O    1.13950  -0.06928  -0.36369                           
  H    0.96608  -0.83110  -0.93042                           
 	Core 	RigidRotor
           SymmetryFactor    1.00000000000000     
 End
  Rotor                             Hindered                 
  Group    7                                                 
  Axis             6           3                             
  Symmetry            3                                      
  Potential[kcal/mol]   4
    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00
  End                                                        
     Frequencies[1/cm]           13
   3846.9      3243.4      3237.7      3100.3      1589.8      1428.8      1427.2      1187.6      911.67      769.31    
   521.17      424.67      416.57    
 ZeroEnergy[kcal/mol]  -0.333034583551029     
  ElectronicLevels[1/cm]                      1
  0.000000000000000E+000   2.00000000000000     
 End
 !*********************************************
 RRHO            !            30
 Geometry[angstrom]            7
  C   -1.23466   0.12184   0.43156                           
  H   -1.19179   0.12194   1.51390                           
  H    0.13507   0.09668   0.03826                           
  H   -1.66738  -0.77758   0.01059                           
  H   -1.61649   1.03792  -0.00204                           
  O    1.13884  -0.06924  -0.36348                           
  H    0.96723  -0.83079  -0.93049                           
 	Core 	RigidRotor
           SymmetryFactor    1.00000000000000     
 End
  Rotor                             Hindered                 
  Group    7                                                 
  Axis             6           3                             
  Symmetry            3                                      
  Potential[kcal/mol]   4
    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00
  End                                                        
     Frequencies[1/cm]           13
   3851.1      3245.4      3239.6      3100.8      1610.9      1427.6      1426.2      1199.0      871.80      743.19    
   526.21      414.73      402.03    
 ZeroEnergy[kcal/mol]   -1.15912491801347     
  ElectronicLevels[1/cm]                      1
  0.000000000000000E+000   2.00000000000000     
 End
 !*********************************************
 RRHO            !            31
 Geometry[angstrom]            7
  C   -1.23572   0.12186   0.43186                           
  H   -1.18962   0.12192   1.51386                           
  H    0.15056   0.09564   0.03327                           
  H   -1.66588  -0.77807   0.00987                           
  H   -1.61460   1.03838  -0.00286                           
  O    1.13824  -0.06920  -0.36329                           
  H    0.96849  -0.83047  -0.93057                           
 	Core 	RigidRotor
           SymmetryFactor    1.00000000000000     
 End
  Rotor                             Hindered                 
  Group    7                                                 
  Axis             6           3                             
  Symmetry            3                                      
  Potential[kcal/mol]   4
    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00    0.00    0.01    0.01    0.00
  End                                                        
     Frequencies[1/cm]           13
   3855.7      3247.5      3241.7      3101.6      1634.9      1426.7      1425.3      1220.0      825.22      716.29    
   534.76      400.48      383.31    
 ZeroEnergy[kcal/mol]   -1.99261970875347     
  ElectronicLevels[1/cm]                      1
  0.000000000000000E+000   2.00000000000000     
 End
 !*********************************************
       Tunneling    Eckart
       ImaginaryFrequency[1/cm]    1095.66000000000     
       WellDepth[kcal/mol]           7.10 
       WellDepth[kcal/mol]          19.98 
 End 
End


