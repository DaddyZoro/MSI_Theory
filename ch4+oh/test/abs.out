Rate Units: cm^3/sec

 T, K REACS->PRODS PRODS->REACS
  300  7.19008e-15  3.52925e-26
  400  3.61904e-14  7.34318e-23
  500  1.13584e-13   8.1822e-21
  600  2.68583e-13  2.04206e-19
  700  5.31626e-13   2.1517e-18
  800  9.33524e-13   1.3181e-17
  900  1.50467e-12  5.60885e-17
 1000  2.27425e-12  1.84475e-16
 1100  3.26971e-12  5.01961e-16
 1200   4.5164e-12  1.18253e-15
 1300  6.03748e-12  2.48942e-15
 1400  7.85399e-12  4.79072e-15
 1500  9.98496e-12  8.57082e-15
 1600  1.24476e-11  1.44374e-14
 1700  1.52573e-11  2.31243e-14
 1800  1.84279e-11  3.54907e-14
 1900  2.19718e-11  5.25176e-14
 2000  2.59002e-11  7.53016e-14
 2100  3.02228e-11  1.05047e-13
 2200  3.49483e-11  1.43058e-13
 2300  4.00847e-11  1.90729e-13
 2400  4.56388e-11  2.49534e-13
 2500  5.16169e-11   3.2102e-13
